packages/quantic/force-app/main/default/lwc/searchBoxStyle/searchBoxStyle.css

```diff
@@ -27,8 +51,9 @@
 
 .searchbox__submit-button {
   width: 46.75px;
-  border-radius: 0;
   height: 48px;
+  border: none;
+  border-radius: 0 var(--lwc-borderRadiusMedium, 0.25rem) var(--lwc-borderRadiusMedium, 0.25rem) 0;
```

Comment:
Not sure about this either, why was this added here? Is it because it was in both css files?